AxonDeepSeg/models/download_model.py

```diff
@@ -14,10 +23,17 @@ def main():
         print(
             "ERROR: Data was not succesfully downloaded and unzipped- please check your link and filename and try again."
         )
-    shutil.move("default_SEM_model", "AxonDeepSeg/models/default_SEM_model")
-    shutil.move("default_TEM_model", "AxonDeepSeg/models/default_TEM_model")
+
+    if sem_destination.exists():
+        print('SEM model folder already existed - deleting old one.')
```

Comment:
The print appears when using `download_models` from `setup.py` directly in the terminal but not while running `pip install -e .`

Reply:
That's normal - we can't print anything when running setup.py (which is what `pip install -e .` does). There's no way around this: https://github.com/pypa/pip/issues/2933#issuecomment-290407667